### ISSUE TITLE
Technic modpack is broken

### ISSUE THREAD

**Issue by Shadotak** [mechaet/ModderationConfigs#44]
Trying to download this modpack on the technic launcher gives an error saying:
"Error unzipping a file for the following pack: Modderation: Permabanned.
Attempting to extract file modderation-permabanned-edition-1.0.zip, but it did not exist.
Please consult the modpack author."

I'd also like to download the server to host on my local network, but the site is down.

**Comment by sandalle** [comment]
I might have a copy of some of those files somewhere for the server, and maybe the client, but I don't believe @mechaet 's Solder server is running anymore to provide those files to Technic.

**Comment by sandalle** [comment]
I've uploaded the server files I have for [Modderation Permabanned 1.1.9 on Dropbox](https://www.dropbox.com/s/tt1q0k76jsdyigu/modderation-permabanned-1.1.9-server.zip?dl=0) (last release), if that helps. I uninstalled Technic and all its managed modpacks last year and don't seem to have any client copies left.

I do have a Minecraft 1.12 modpack, [NST World of Color](https://minecraft.curseforge.com/projects/nst-world-of-color) on CurseForge, if that's what you're looking for.
